kits/ci-cd-diagnosis-agent/.env.example

```
@@ -0,0 +1,7 @@
+CICD_DIAGNOSIS_FLOW_ID=
+LAMATIC_API_KEY=
+LAMATIC_API_URL=
+LAMATIC_PROJECT_ID=
+GITHUB_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_CLIENT_ID key should go before the LAMATIC_API_KEY key

(UnorderedKey)